unity-renderer/Assets/DCLServices/WearablesCatalogService/LambdasWearablesCatalogService.cs

```diff
@@ -175,20 +205,19 @@ public async UniTask<(IReadOnlyList<WearableItem> wearables, int totalAmount)> R
 
         public async UniTask<IReadOnlyList<WearableItem>> RequestBaseWearablesAsync(CancellationToken ct)
         {
-            var serviceResponse = await lambdasService.Get<WearableWithoutDefinitionResponse>(
-                NON_PAGINATED_WEARABLES_END_POINT,
-                NON_PAGINATED_WEARABLES_END_POINT,
-                REQUESTS_TIME_OUT_SECONDS,
-                urlEncodedParams: ("collectionId", BASE_WEARABLES_COLLECTION_ID),
-                cancellationToken: ct);
+            var url = $"{catalyst.contentUrl}entities/active/collections/{BASE_WEARABLES_COLLECTION_ID}";
+
+            var request = await lambdasService.GetFromSpecificUrl<WearableCollectionResponse>(url, url, cancellationToken: ct);
 
-            if (!serviceResponse.success)
+            if (!request.success)
                 throw new Exception("The request of the base wearables failed!");
 
-            MapLambdasDataIntoWearableItem(serviceResponse.response.wearables);
-            AddWearablesToCatalog(serviceResponse.response.wearables);
+            var wearables = request.response.entities.Select(dto => dto.ToWearableItem(catalyst.contentUrl, assetBundlesUrl)).ToList();
```

Comment:
I just checked and it looks like the return value is not used anywhere. Then I wonder if we can just use pools to process this huge mutation and don't expose it?